src/integrations/settings-integration.php

```diff
@@ -428,6 +433,7 @@ protected function get_script_data() {
 			'taxonomies'                     => $transformed_taxonomies,
 			'fallbacks'                      => $this->get_fallbacks(),
 			'showNewContentTypeNotification' => $show_new_content_type_notification,
+			'isBlackFriday'                  => YoastSEO()->classes->get( Promotion_Manager::class )->is( 'black-friday-2023-promotion' ),
```

Comment:
If you would add this to the other `isX` things in the `preferences`, you would also be able to use the `selectPreference` selector. Instead of needing to use the window object directly in the component.